graphql/util.go

```diff
@@ -514,6 +505,23 @@ func removeGeneralSubscriptions(usr *user.DBUser, subscriptions []event.Subscrip
 	return filteredSubscriptions
 }
 
+func makePatchDuration(t, m string) *PatchDuration {
+	var tPointer *string
```

Comment:
Rather than defining tPointer and mPointer, can we have a res := &PatchDuration{} and just set res.Makespan and res.TimeTaken to &t and &m directly?

Reply:
Very much yes :) 
Thanks! Too much copy/pasting.